app/Policies/BoardPolicy.ts

```diff
@@ -0,0 +1,16 @@
+import { BasePolicy } from '@ioc:Adonis/Addons/Bouncer'
+import User from 'App/Models/User'
+import Board from 'App/Models/Board'
+
+export default class BoardPolicy extends BasePolicy {
+  public async viewList(user: User) {}
+  public async view(user: User, board: Board) {}
```

Comment:
```suggestion
  public async viewList(user: User) {}
  public async view(user: User, board: Board) {}
```

please, delete unused polices